[MDB IGNORE] Karmac Map Pack #1 [OWP Mackay Station + ONI Digsite 451]

KKarmac opened the pull request:
# About the pull request

A recreation of Cairo Station from Halo 2 with some creative liberties taken to fill out the place. Available as a ground or shipmap.

Lore-wise this is a Moncton-Class Orbital Weapon Platform, scaled down a bit, and only portraying one side of the station's habitat ring. I have no set planet in mind for it to be stationed over, and in-universe these would either be part of a battle-cluster of OWP's or uncommonly seen in smaller groups of 2-3 over a planet, or rarely operating on its own.

Now also including a SWORD base inspired map; ONI Digsite, so you don't have to run Blacksite just to have a secret facility type map. It has no set purpose, and is perfect for GM's that want a secret base with secret things happening inside of it. It consists of two levels; one slightly below ground section with many security checkpoints and mostly intended as a traversal point to get to the more important lower levels, the second level is far below ground and built into a massive ice cave system, with a handful of specific building POI's.

Additionally, includes two new props; a status display screen that says UNSC (at some point I may choose to be less lazy and have this flicker or show something else), and a nuclear bomb re-sprited to resemble the halo 2 covenant bomb.

# Explain why it's good for the game

I think it's awesome, should be good for a bunch of different ops, or just sitting around in. 

A few important notes, until pelicans are added there's no functional LZ on the map, so make sure to put one in yourself if you need it. This thing sucks up power like nobody's business while the generators aren't running, and it's currently a ground map only deal so the generators will be broken on roundstart, expect power to cut out across sectors within 10-20 minutes. The exterior spacewalk areas do not currently harm players, imagine them as being enclosed by glass or something to explain that away if anyone asks, though as a point you can still go careening into the void of space, I've just made it hard to do that on accident.

The Digsite map has no functional issues at present, but has had a problem with some blast doors cycling visually without actually entering the 'open' state, meaning players couldn't see or pass through them. I could not recreate this, so I'm assuming it was a freak incident. As before, it also has no functional LZ, so make sure to add one in when you use it. 

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

https://imgur.com/a/idtn2jg

https://imgur.com/4PuYy7c

</details>


# Changelog

:cl: Karmac
mapadd: Mackay Station (Ground & Shipmap versions), ONI Digsite 451
add: prop status display for the unsc, covenant bomb
/:cl:

KKarmac committed:
adds .dmm map file

KKarmac committed:
adds most relevant map code

KKarmac committed:
fixes area parents

KKarmac committed:
adds more intercoms

KKarmac committed:
adds elevator code

KKarmac committed:
adds shuttles and adjusts elevator code

KKarmac committed:
elevator related mapping adjustments

KKarmac committed:
fixes elevator coding errors and missing map path

KKarmac committed:
adjusts elevator code fuck-up

KKarmac committed:
fixes error with map file pathing

KKarmac committed:
almost solves elevator related issues

KKarmac committed:
maybe solves reserved transit issue

KKarmac committed:
solves elevators, adds missing turfs under stairs

KKarmac committed:
adds more landing lights to the hangar

KKarmac committed:
reduces blast doors, adds controls for them

cm13-github commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 09/28/2025 20:11:24 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 5f7e7f63c2e6712e5183435ec630f2183f5b9fcb
Pull Request: ee98ead8d141ddf9804bfe6a8f4c9ca94f666439
Server: 37ce40c1d6687e190266aa8e911e9dde10c75184
</details>
<details><summary>Test Merge Updated @ 09/29/2025 22:28:42 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 5f7e7f63c2e6712e5183435ec630f2183f5b9fcb
Pull Request: 480e32475b6987e74992b79550700400ccf14a2d
Server: dd60f760855ec85bdc2d030e6294234bc259770c
</details>
<details><summary>Test Merge Removed @ 10/02/2025 20:19:00 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 10/03/2025 04:22:00 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
SpartanBobby

##### Revision
Origin: 5f7e7f63c2e6712e5183435ec630f2183f5b9fcb
Pull Request: 77cb2dd98267cfbc274dfebe2cb3cbe643041c25
Server: f96d996d4b889b5825000335daa3aedf48953b77
</details>
<details><summary>Test Merge Removed @ 10/30/2025 02:15:20 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 12/23/2025 11:38:20 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: be2f27eb8ab022ec1b41e8187e33683553efd7e7
Server: 834cb221cc255973dac8c6094674316f05d90c65
</details>
<details><summary>Test Merge Updated @ 12/23/2025 11:48:06 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 29fd8ac7bfec8acad26430e487ddd90808c5d52b
Server: 7e6d68200bec2ccf0809f10b049af8d729522d11
</details>
<details><summary>Test Merge Updated @ 12/23/2025 12:09:36 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 759c6251846b677e5dc1239a6fa2195b4fd45a49
Server: af425d324bc2eb35c9b9696e6db2db7483137685
</details>
<details><summary>Test Merge Updated @ 12/23/2025 12:15:01 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 0b1f55096efaef15228b90292431288096d58b5d
Server: e063fd810fd9d9155d81622051cf43c23a94667b
</details>
<details><summary>Test Merge Updated @ 12/23/2025 12:24:02 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 2e616762a3e855cfc762e1982fe4669d80298c78
Server: 9588d870f44b3d0408befd34cd973caa7d6a77e4
</details>
<details><summary>Test Merge Updated @ 12/23/2025 12:39:15 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: f9d8b6e8f5185fb620ecabe60381b8c94e3e4d24
Server: 1b974777c2ca810a343a0491b5ed2b4306bc6af8
</details>
<details><summary>Test Merge Updated @ 12/23/2025 12:54:35 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 1f0800c4acb050d9e1c6069792d7d40e253bb120
Server: 403bc2268a452cb37af60aeaa58616808baf962c
</details>
<details><summary>Test Merge Updated @ 12/23/2025 13:10:52 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 1799b7237e30acab7ade004cb42b2f1868ca2d3d
Server: c6f4eaf59f88fdb89548d8df9722f21341f949b2
</details>
<details><summary>Test Merge Updated @ 12/24/2025 13:55:23 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 138e9df23109e13feeab3b4b77bc7f4e213e512e
Pull Request: 29ea99a26f90c9a57bc55d58fe5bd8ec640fb4e0
Server: 032dd6d3869986130b163d1c8fdb9170e05fdaad
</details>
<details><summary>Test Merge Removed @ 01/05/2026 11:48:05 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 01/05/2026 12:33:24 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 633eb8929a23fb8d2803bfa8ab88607b734c194a
Server: 91d656cef0978d73de6537e7fe6e9096b5067592
</details>
<details><summary>Test Merge Updated @ 01/10/2026 08:04:24 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 2781bd4a8ea0a4f282f564acc58b75dd020219dc
Server: d9b611d9d20ada4c22aea04cf402e441e168796e
</details>
<details><summary>Test Merge Updated @ 01/10/2026 08:19:25 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: d27268a57beb4b26af8d1b083393f0eb93ed3cb4
Server: 2c5079acacb923e778a5372fad00c90b57bc145e
</details>
<details><summary>Test Merge Updated @ 01/13/2026 07:26:58 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 3cfc4f3e758dd721a920628e3f8a1fc3507109c6
Server: 204143af47e552f70c0cd6bc3f6e6f5d0931f3cf
</details>
<details><summary>Test Merge Updated @ 01/14/2026 00:05:47 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: dc862b8ed040575b3e7b904966ab4eb704ec49e8
Server: 7e5737af213764a8060b766e2512f30ca93e7699
</details>
<details><summary>Test Merge Updated @ 01/16/2026 22:33:11 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 0a582502f84e3d2151365b31fb74d560033737b6
Server: b3bf7872838876ba6b5d115b0bc46265a7c6d99e
</details>
<details><summary>Test Merge Updated @ 01/16/2026 23:04:25 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: a6544d42eb15a1a6b6413c8518528f0a5e7fdb80
Server: 76a31f6c65915b2b52eee3a7b0f34bfeb034717f
</details>
<details><summary>Test Merge Updated @ 01/16/2026 23:26:59 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: f4fe03b32d3a47de6e6b50be5dbbd734a6f9f34c
Server: b608862f2f8b74bc4fddf1de251c3240d376c557
</details>
<details><summary>Test Merge Updated @ 01/16/2026 23:40:06 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 640efd0dc0d1cef77cbadc25797f3fa4304214d4
Server: bf8243288b60ce35c415ae1889dd5d8920fa92cf
</details>
<details><summary>Test Merge Updated @ 01/17/2026 03:29:11 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 4f16f5e9bb109ce2a15c0b7c78c9e32c1ffe7648
Pull Request: 1f577597182f6d8ad3c03dd7ec0cfaeccbcdf5e3
Server: b90a990f82a091e2b69b0e24e5523653e316218f
</details>
<details><summary>Test Merge Updated @ 01/17/2026 10:47:30 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 1bea42ec97bf17860b1bef06f65d778618145bf8
Pull Request: 1f577597182f6d8ad3c03dd7ec0cfaeccbcdf5e3
Server: 234f76e3329ec9d6761725b64ed176433bd7fed9
</details>
<details><summary>Test Merge Updated @ 01/17/2026 11:41:10 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 1bea42ec97bf17860b1bef06f65d778618145bf8
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: 04d875432299a48af6a8c87d3b8568cf3578d6f6
</details>
<details><summary>Test Merge Updated @ 01/18/2026 07:45:10 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
ody_sseus

##### Revision
Origin: dc38691ffb7928cd8c891c14626b73f572981a47
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: 5363a5be531d7fa0205615287051a6e9ab437e5d
</details>
<details><summary>Test Merge Updated @ 01/18/2026 08:27:24 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
ody_sseus

##### Revision
Origin: dc38691ffb7928cd8c891c14626b73f572981a47
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: e905ade9706598462704dd52aeb5400fa7807748
</details>
<details><summary>Test Merge Updated @ 01/18/2026 08:34:39 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
ody_sseus

##### Revision
Origin: dc38691ffb7928cd8c891c14626b73f572981a47
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: 898d0500ac60fa8a20a40744dbdd9f611ffaf06b
</details>
<details><summary>Test Merge Updated @ 01/18/2026 08:40:48 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
ody_sseus

##### Revision
Origin: dc38691ffb7928cd8c891c14626b73f572981a47
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: 798ccc148330978ad7848209ba955b473ace44f9
</details>
<details><summary>Test Merge Updated @ 01/19/2026 02:54:17 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 93bf5a06de25a87d4f830b61b9888a4ea62350f2
Pull Request: 9e4477c73a25d4ae9f35b91f54b779d238a61a33
Server: a21a5676fdea8c48daef477c91f3fb64fab3e21f
</details>
<details><summary>Test Merge Updated @ 01/21/2026 10:12:20 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 93bf5a06de25a87d4f830b61b9888a4ea62350f2
Pull Request: 0b1787708d18a04464449b7b4bd38f826f9b0ee1
Server: 2dd82dfca322d9dfbb1e70c5840547c486575e41
</details>
<details><summary>Test Merge Updated @ 01/25/2026 04:43:27 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 93bf5a06de25a87d4f830b61b9888a4ea62350f2
Pull Request: 0b1787708d18a04464449b7b4bd38f826f9b0ee1
Server: 4dab8d36ae87d680182e0ed9ae13223f1d8e636a
</details>
<details><summary>Test Merge Updated @ 02/03/2026 18:24:03 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: f227ef281c6580e26e665bc191f77e4cc2c2b986
</details>
<details><summary>Test Merge Updated @ 02/05/2026 01:59:40 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: 4921b6831a836120ee976c537d3a5f6c688fd31d
</details>
<details><summary>Test Merge Updated @ 02/06/2026 17:43:48 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: d80bb0b06986f3530ac3d25d48eb67ff4fbf3ddc
</details>
<details><summary>Test Merge Updated @ 02/06/2026 18:04:47 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: 60820ee430721e0a9d69bf0172327f78986543b4
</details>
<details><summary>Test Merge Updated @ 02/06/2026 18:06:54 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: d80bb0b06986f3530ac3d25d48eb67ff4fbf3ddc
</details>
<details><summary>Test Merge Removed @ 02/06/2026 18:15:24 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 02/06/2026 19:31:37 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 752f940da43669b62e483ce4451deecfaf4de064
Pull Request: b22f392501b9c275a9f123c028ff938c27a50e8f
Server: 4921b6831a836120ee976c537d3a5f6c688fd31d
</details>
<details><summary>Test Merge Removed @ 02/16/2026 02:53:19 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 03/12/2026 00:27:37 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 5969b4a0dfced922ee4a98bf625faee3bd104233
Server: 5a8146665362d7b73015788beda45f284d8e296d
</details>
<details><summary>Test Merge Updated @ 03/12/2026 05:45:49 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 8e92f3c0148b19aed307e03244b533364b2cb2aa
Server: f90ff11bca4e082afe56f7c63a98b604ffc48646
</details>
<details><summary>Test Merge Updated @ 03/16/2026 03:13:27 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 51cfcb16cf8391bc688e9b576496fdd16a233d31
Server: 78b211df0d57dd7347414b4e64563de685943b6a
</details>
<details><summary>Test Merge Updated @ 03/16/2026 03:26:43 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 2c1f66c9a859aa1d3d629102bd29529855f9cd8e
Server: a47a80683e839cd8e1bcb09a65a97d575bfa9acc
</details>
<details><summary>Test Merge Updated @ 03/16/2026 03:35:44 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 3960cf0719ab69d2afce1bf10788bcce07c1f91a
Server: 513d3514330cc74850388607f9faabbd3e2568bc
</details>
<details><summary>Test Merge Updated @ 03/16/2026 03:53:32 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: eee2b58bf6d4f4cef17c3b8f6518e377495f16fb
Server: 91e28440ec8d3724d690bb0c2ec45345d4f5b21d
</details>
<details><summary>Test Merge Updated @ 03/16/2026 04:03:09 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 95a84ab9f59f9118e5543f664b2793e7a1841c55
Pull Request: 695ab25a94201c56c0262020940ef587c2560d4b
Server: f9c4a5f4e9576d45b308832454ecf30141b01c0a
</details>
<details><summary>Test Merge Removed @ 03/27/2026 04:54:23 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 03/28/2026 02:53:32 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 3e34a778f2c9a81b8b54250936073ad27b7cc707
Pull Request: 561f27d4559cc4d74a757c553ba01292148ccc0d
Server: 86d1c9b3dbb9244d98b292769b0bd188c6f4828a
</details>
<details><summary>Test Merge Updated @ 04/02/2026 13:56:28 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
GDSPathe

##### Revision
Origin: 3e34a778f2c9a81b8b54250936073ad27b7cc707
Pull Request: 561f27d4559cc4d74a757c553ba01292148ccc0d
Server: ffc160c5881ecb841f1ce64a3893cdeec9fdd93e
</details>
<details><summary>Test Merge Updated @ 04/03/2026 04:57:31 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 3e34a778f2c9a81b8b54250936073ad27b7cc707
Pull Request: 561f27d4559cc4d74a757c553ba01292148ccc0d
Server: 8e951511af99cfd35bbf728e1e957d9bdbb52beb
</details>
<details><summary>Test Merge Updated @ 04/04/2026 00:27:07 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Verbs

##### Revision
Origin: 3e34a778f2c9a81b8b54250936073ad27b7cc707
Pull Request: 561f27d4559cc4d74a757c553ba01292148ccc0d
Server: 692f2743ff2ae6763cae3bfcea19cc84f902e583
</details>
<details><summary>Test Merge Updated @ 04/05/2026 08:23:55 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 3e34a778f2c9a81b8b54250936073ad27b7cc707
Pull Request: 561f27d4559cc4d74a757c553ba01292148ccc0d
Server: dfcdc93d7f0e7aa80b29d7418a5f2c65a1729fb9
</details>
<details><summary>Test Merge Removed @ 04/05/2026 08:29:11 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 04/18/2026 12:03:30 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
humiliatedgoblin

##### Revision
Origin: de3018e3d069569d7afc634763d6036139b8f76b
Pull Request: 59a70266ce449cf177d985651f36e5690ed69340
Server: c129cc3fe7cf46941e69e24446f7ad03bd80d605
</details>
<details><summary>Test Merge Removed @ 04/18/2026 12:09:37 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 04/22/2026 06:24:00 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 33a011138b2529982de18896616a7cfa9d38f376
Pull Request: ac173c701d98193b985bbbfe9dafa82012385383
Server: ab6cf48b4cee5dc41a0c6d9b68c20cb5eb844d89
</details>
<details><summary>Test Merge Updated @ 04/23/2026 13:29:18 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Karmac

##### Revision
Origin: 33a011138b2529982de18896616a7cfa9d38f376
Pull Request: 5d6398ae32a51c93157de05d9fce0b8060f4be7d
Server: 6c5aa2e6fa1d31169991c893444c8ef45aa609c9
</details>
<details><summary>Test Merge Updated @ 04/25/2026 22:18:22 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
Dumbgunner

##### Revision
Origin: 33a011138b2529982de18896616a7cfa9d38f376
Pull Request: 8c4697c6f05dfeafa13a9621547c964f60095339
Server: 576d691c1490a52c12547e9f3c8b20e2f98e9d82
</details>
<details><summary>Test Merge Removed @ 04/25/2026 22:24:39 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 04/26/2026 19:53:00 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
runner

##### Revision
Origin: 33a011138b2529982de18896616a7cfa9d38f376
Pull Request: 8c4697c6f05dfeafa13a9621547c964f60095339
Server: 5ea347e5856a175c5d062c116a6fc5eec2d71690
</details>
<details><summary>Test Merge Removed @ 04/29/2026 06:29:28 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>
<details><summary>Test Merge Deployed @ 04/30/2026 22:24:07 +00:00:</summary>

##### Server Instance
PVECM: Halo

##### Merged By
runner

##### Revision
Origin: a4dcbc9dff4f4e9141b6db3c64f5b886b6671d16
Pull Request: 8c4697c6f05dfeafa13a9621547c964f60095339
Server: cd51629ca907c412b1cba436b60cdfe31bfbd488
</details>
<details><summary>Test Merge Removed @ 04/30/2026 22:32:14 +00:00:</summary>

##### Server Instance
PVECM: Halo
</details>

KKarmac committed:
maybe fixes locking elevators and upgrades apcs

KKarmac committed:
elevator nonsense + space railings

KKarmac committed:
names a few airlocks

KKarmac committed:
adjusts prop layers and light placements

KKarmac committed:
minor adjustments to light placement again

KKarmac committed:
makes some airlocks seethrough

KKarmac committed:
minor map edits and layering adjustments

KKarmac committed:
adjusts lighting, decals, and elevator direction

KKarmac committed:
more minor mapping adjustments

KKarmac committed:
changes some hull walls to regular walls

KKarmac committed:
more decal editing

KKarmac committed:
adds prop to replace standard display screen

KKarmac committed:
fixes prop to not use power

KKarmac committed:
replaces real status displays with props

KKarmac committed:
adds more floor decals we love floor decals

KKarmac committed:
adds c-bomb code, untested, needs icon sprites

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'master' into mackay_station

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
fixes an error with a map define i somehow made

KKarmac committed:
removes c-bomb to check if it fixes other issue

KKarmac committed:
readds c-bomb because it wasnt the problem

KKarmac committed:
fixes an extra line being in maps.dm for no reason

KKarmac committed:
adds marine shipmap version & slight armory edit

KKarmac committed:
adds weather code to groundmap ver just in case

KKarmac committed:
adds personal lockers

KKarmac committed:
slight medbay lighting edit

KKarmac committed:
attempts to make tram doors not lock on arrival

KKarmac committed:
tries to fix groundmap not appearing

KKarmac committed:
fix groundmap issue part 2

KKarmac committed:
messing with elevators again

KKarmac committed:
surely it will work this time

KKarmac committed:
still fiddling with elevators

KKarmac committed:
fixes elevator doors locking on arrival, surely

KKarmac committed:
elevators i love them

KKarmac committed:
testing if i am stupid

KKarmac committed:
further testing

KKarmac committed:
adds map tables to the bridge on both versions

KKarmac committed:
elevator stuff still not working how i want it to

KKarmac committed:
makes station trams/elevators come and go quicker

KKarmac committed:
ACTUALLY makes the elevators work now

KKarmac committed:
adds t-comms area to shipmap cos i forgot it

KKarmac committed:
fixes the hangar elevator to work now also

KKarmac committed:
removes some legacy sprites

KKarmac committed:
incredibly minor map edit

KKarmac committed:
slight map edit to remove a layering issue

KKarmac committed:
adds ONI Digsite 451 and minor station map edits

KKarmac committed:
fixes ONI map elevator

KKarmac committed:
slight map edits to fix lockers and lights

KKarmac committed:
adds areas, door/button names, elevator fixes

KKarmac committed:
removes var-edits from area and fixes its code

KKarmac committed:
fixes barricade health and turret lore

KKarmac committed:
adds observer landmark to shipmap

KKarmac committed:
adjusts areas, adds apcs, fixes computer facing

KKarmac committed:
adjusts computers again

KKarmac committed:
fixes water in the cave

KKarmac committed:
map edit + fixes turfs and weather event issue

KKarmac committed:
density fix + ambience

KKarmac committed:
slight map edits

KKarmac committed:
slight map edits v2

KKarmac committed:
adds a few missing lights

KKarmac committed:
removes unnecessary lights

KKarmac committed:
adds lights to a single checkpoint missing them

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
adds covenant_bomb as a nuclearbomb subtype

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'master' into mackay_station

KKarmac committed:
ONI digsite map tweaks

KKarmac committed:
digsite map tweaks pt 2

KKarmac committed:
Merge branch 'mackay_station' of https://github.com/KKarmac/cmss13-pve-halo-mapping into mackay_station

KKarmac committed:
mackay station map tweaks & elevator adjusts

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
adds ma5b vendors to mackay station maps

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Update oni_digsite_451.dmm

KKarmac committed:
Update oni_digsite_451.dmm

KKarmac committed:
fixes Mackay Station shipmap and regular ver

KKarmac committed:
adjusts shipmap spawns

KKarmac committed:
fixes shipmap spawns, workaround for landmark bug

KKarmac committed:
adjusts personal lockers

KKarmac committed:
attempt to fix personal lockers

KKarmac committed:
adjusts button location

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
personal weapon spawner added to shipmap armory

KKarmac committed:
trams are no longer nsfw

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
adds LZs to all maps

KKarmac committed:
removes spacewalk, adjusts armory content

KKarmac committed:
part 2 of armory adjustment

KKarmac committed:
adds lights that i forgot

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

BonniePandora commented on maps/map_files/ONI_Digsite_451/oni_digsite_451.dmm:
```diff
@@ -0,0 +1,3195 @@
+"ffO" = (/obj/structure/platform/strata,/obj/structure/platform/strata{dir = 8},/turf/open/auto_turf/snow/layer0,/area/oni_digsite/sub_level_9/snow)
+"ffP" = (/obj/structure/machinery/cm_vending/sorted/tech/electronics_storage,/turf/open/floor/prison,/area/oni_digsite/sub_level_9/small_buildings)
+"fgj" = (/obj/structure/machinery/camera/autoname{dir = 8},/turf/open/floor/dark,/area/oni_digsite/sub_level_2/south/checkpoint_alpha)
+"fgM" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened{dir = 8},/obj/item/reagent_container/food/drinks/coffeecup{pixel_y = 12; pixel_w = 8},/turf/open/floor/prison/floor_plate,/area/oni_digsite/sub_level_2/north/offices)
```
```suggestion
"fgM" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened{dir = 8},/obj/item/reagent_container/food/drinks/coffeecup{pixel_y = 12; pixel_x = 8},/turf/open/floor/prison/floor_plate,/area/oni_digsite/sub_level_2/north/offices)
```

BonniePandora commented on maps/map_files/ONI_Digsite_451/oni_digsite_451.dmm:
```diff
@@ -0,0 +1,3195 @@
+"fPn" = (/obj/structure/prop/unsc_crate/big{pixel_x = 2; pixel_y = 2},/turf/open/floor/prison,/area/oni_digsite/sub_level_9/command)
+"fPZ" = (/obj/structure/barricade/handrail/strata,/obj/structure/pipes/vents/pump,/turf/open/floor/prison/bright_clean,/area/oni_digsite/sub_level_9/small_buildings)
+"fQv" = (/obj/structure/reagent_dispensers/watertank/unsc{desc = "An olive drab barrel containing water. Better not mix it up with the hydrogen tank. Someone put this one in the canteen, just in case you're tasting metal flavored water as opposed to metal and plastic flavored water."; pixel_x = 8; pixel_y = 14},/obj/structure/reagent_dispensers/watertank/unsc{pixel_x = -2},/turf/open/floor/prison ...
+"fQG" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened,/obj/structure/window/reinforced/toughened{dir = 8},/obj/item/reagent_container/food/drinks/coffeecup{pixel_y = 12; pixel_w = 8},/turf/open/floor/prison/floor_plate,/area/oni_digsite/sub_level_2/north/offices)
```
```suggestion
"fQG" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened,/obj/structure/window/reinforced/toughened{dir = 8},/obj/item/reagent_container/food/drinks/coffeecup{pixel_y = 12; pixel_x = 8},/turf/open/floor/prison/floor_plate,/area/oni_digsite/sub_level_2/north/offices)
```

BonniePandora commented on maps/map_files/ONI_Digsite_451/oni_digsite_451.dmm:
```diff
@@ -0,0 +1,3195 @@
+"mwf" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened,/obj/item/folder/white{pixel_x = -8},/obj/item/tool/pen{pixel_x = -5; pixel_y = 4},/obj/item/paper_bin/uscm{pixel_x = 8; pixel_y = 6},/turf/open/floor/prison,/area/oni_digsite/sub_level_9/research)
+"mxB" = (/obj/structure/machinery/light/double/blue{dir = 4},/turf/open/floor/almayer/mono,/area/oni_digsite/sub_level_2/south)
+"mxD" = (/obj/structure/barricade/handrail/strata,/obj/effect/decal/cleanable/dirt{layer = 2.52},/turf/open/floor/prison/bright_clean,/area/oni_digsite/sub_level_9/small_buildings)
+"mxP" = (/obj/structure/platform/strata{dir = 4},/obj/structure/platform/strata,/turf/open/auto_turf/snow/layer2,/area/
```
```suggestion
"nIT" = (/obj/structure/surface/table/almayer{color = "#96918c"},/obj/structure/window/reinforced/toughened{dir = 8},/obj/structure/window/reinforced/toughened,/obj/item/reagent_container/food/drinks/coffeecup{pixel_y = 12; pixel_x = 8},/turf/open/floor/prison/floor_plate,/area/oni_digsite/sub_level_2/north/offices)
```

BonniePandora reviewed (changes requested):
Besides the three weird vars on placed items, just needs the mapmerge tool ran for the formatting update and these should be good to go.

KKarmac committed:
Fixup maps in TGM format

new         : maps/map_files/Mackay_Station/Mackay_Station.dmm
new         : maps/map_files/Mackay_Station_Shipmap/Mackay_Station_Shipmap.dmm
new         : maps/map_files/ONI_Digsite_451/oni_digsite_451.dmm
new         : maps/shuttles/digsite_elevator.dmm
new         : maps/shuttles/stationfirecontrol.dmm
new         : maps/shuttles/stationhangar.dmm
new         : maps/shuttles/stationtrambravo.dmm
new         : maps/shuttles/stationtramgolf.dmm
new         : maps/shuttles/stationtramromeo.dmm

Automatically commited by: tools\mapmerge2\fixup.py

KKarmac committed:
Merge branch 'cmss13-devs:master' into mackay_station

KKarmac committed:
Merge branch 'mackay_station' of https://github.com/KKarmac/cmss13-pve-halo-mapping into mackay_station

KKarmac committed:
fixes map formatting, removes wumbo

KKarmac committed:
hopefully fixes linters

KKarmac committed:
Merge branch 'master' into mackay_station

KKarmac committed:
fixes linters part 2

KKarmac committed:
Merge branch 'mackay_station' of https://github.com/KKarmac/cmss13-pve-halo-mapping into mackay_station

KKarmac committed:
fixes linters part 3

KKarmac committed:
might fix weather holder issue

DexterDude committed:
Merge branch 'master' into mackay_station
